WTWSMS/decisions/employment_decisions.txt

```diff
@@ -135,7 +144,10 @@ decisions = {
 				hidden_tooltip = {
 					if = {
 						limit = {
-							religion_group = indian_group
+							OR = {
+								religion_group = indian_group
+								religion = hepthal_religion
+						}
 						}
 						add_trait = kshatriya
 						character_event = { id = RoI.30121 }
```

Comment:
The kshatriya, vaishya and brahmin traits have not been adapted to the Heptal religion, and from my understanding they didn't have castes.